pkgs/servers/hylafaxp/default.nix

```diff
@@ -0,0 +1,290 @@
+      --replace "/usr/lib/aliases" "/etc/aliases"
+    substituteInPlace configure \
+      --replace "60002" "65534" \
+      --replace "/usr/local/share/ghostscript/fonts" "${ghostscript}/share/ghostscript/fonts"
+  '';
+
+  hyla_conf = writeText "hyla.conf" ''
+    # Warning, FontMap/FontPath was automatically generated by NixOS
+    FontMap:   @out@/var/spool/hylafax/etc
+    FontPath:  ${ghostscript}/share/ghostscript/fonts:${ghostscript}/share/ghostscript/${ghostscript.version}/Resource/Init:${ghostscript}/share/ghostscript/${ghostscript.version}/lib:${ghostscript}/share/ghostscript/${ghostscript.version}/Resource/Font:${ghostscript}/share/ghostscript/fonts:${ghostscript}/share/fonts/default/ghostscript:${ghostscript}/share/fonts/default/Type1:${ghostscript}/share/fonts/default/TrueType
+  '';
+
+  setup_cache = writeText "setup.cache" ''
+    # Warning, setup.cache was automatically generated by NixOS
+    AWK='${gawk}/bin/awk'
+    BIN='@out@/bin'
+    CAT='${coreutils}/bin/cat'
+    CHGRP='${coreutils}/bin/chgrp'
+    CHMOD='${coreutils}/bin/chmod'
+    CHOWN='${coreutils}/bin/chown'
+    CP='${coreutils}/bin/cp'
+    DPSRIP='ps2fax.exe'
+    ECHO='${coreutils}/bin/echo'
+    ENCODING='base64'
+    FAXQ_SERVER='yes'
+    FILECMD='${file}/bin/file'
+    FONTPATH='${ghostscript}/share/ghostscript/fonts:${ghostscript}/share/ghostscript/${ghostscript.version}/Resource/Init:${ghostscript}/share/ghostscript/${ghostscript.version}/lib:${ghostscript}/share/ghostscript/${ghostscript.version}/Resource/Font:${ghostscript}/share/ghostscript/fonts:${ghostscript}/share/fonts/default/ghostscript:${ghostscript}/share/fonts/default/Type1:${ghostscript}/share/fonts/default/TrueType'
+    FUSER='${psmisc}/bin/fuser'
+    GREP='${gnugrep}/bin/grep'
+    GSRIP='${ghostscript}/bin/gs'
+    HFAXD_SERVER='yes'
+    HFAXD_SNPP_SERVER='no'
+    IMPRIP='psrip'
+    LIBDATA='@out@/etc/hylafax'
+    LIBEXEC='@out@/bin'
+    LN='${coreutils}/bin/ln'
+    MANDIR='@out@/share/man'
+    MIMENCODE='${coreutils}/bin/base64'
+    MKFIFO='${coreutils}/bin/mkfifo'
+    MV='${coreutils}/bin/mv'
+    PATHEGETTY='egetty'
+    PATHGETTY='${utillinux}/bin/agetty'
+    PATH='${coreutils}/bin:${gnused}/bin:${gnugrep}/bin:${gawk}/bin:/bin:/usr/bin:/etc'
+    PATHVGETTY='vgetty'
+    PCL6CMD='pcl6'
+    PSPACKAGE='gs'
+    RM='${coreutils}/bin/rm'
+    SBIN='@out@/bin'
+    SCRIPT_SH='/bin/sh'
+    SED='${gnused}/bin/sed'
+    SENDMAIL='/var/setuid-wrappers/sendmail'
+    SPOOL='@out@/var/spool/hylafax'
+    SYSVINIT='''
+    TARGET='${_arch}-unknown-${_os}-gnu'
+    TIFF2PDF='${libtiff}/bin/tiff2pdf'
+    TIFFBIN='${libtiff}/bin'
+    TTYCMD='${coreutils}/bin/tty'
+    UUCP_LOCKDIR='@out@/var/lock/uucp'
+    UUCP_LOCKTYPE='ascii'
+    UUENCODE='${sharutils}/bin/uuencode'
+  '';
+
+  setup_modem = writeText "setup.modem" ''
+    # Warning, setup.modem was automatically generated by NixOS
+    prompt()
+    {
+       ${coreutils}/bin/echo -n "$* "
+    }
+          ttyPort()
```

Comment:
@Mic92 it's the indentation faxsetup outputs. Same goes for hyla.conf, setup.cache and setup.modem.

I'd rather not touch them for diffing purposes if a future update changes them.